### PR TITLE
RaghvendraT_Assignment_13

### PR DESCRIPTION
**Pull request by RaghvendramaniTOct2020**
image

### CODE REVIEW

File: src/raghvendra/BankOperationUsingSwitch.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+			debitAmount=sc.nextDouble();
+		}
+	
+		bankSwitch(bankName, operation, creditAmount, debitAmount);
+		bankBalance(bankName);
+		bankCreditDebit();
+	}
+	
+
+	public void bankSwitch(String bankName,String operation,double creditAmount,double debitAmount) {
```
Comment (harshvegada) [inline src/raghvendra/BankOperationUsingSwitch.java:52]: Hi @RaghvendramaniTOct2020 
Here think on real time scenario when we are going to bank for credit operation, that time bank not asking customer to do debit as well right?

In this method, it give impression like above scenario.

**Review by harshvegada — CHANGES_REQUESTED** [review]
Look like you missed individual bank credits & debit operations!
Pls add those methods in class

File: src/raghvendra/BankOperationUsingSwitch.java (hunk 1 above)
Comment (RaghvendramaniTOct2020) [reply src/raghvendra/BankOperationUsingSwitch.java]: incorporated the changes suggested. Thanks

**Comment by RaghvendramaniTOct2020** [comment]
> Look like you missed individual bank credits & debit operations!
> Pls add those methods in class

incorporated changes in new commit..pls review

**Review by harshvegada — APPROVED** [review]
Thanks for Quick Change @RaghvendramaniTOct2020 
Good to merge!!

**Pull request merged**